### ISSUE TITLE
Collapse lanes horizontally

### ISSUE THREAD

**Issue by harshaktg** [rcdexta/react-trello#410]
**Difficult to scroll horizontally when there are a lot of lanes**
In complex applications, we often have a large number of lanes, and sometimes it's error-prone that we drag and drop to another lane. Right now, we can collapse lanes vertically. But, It'll be better if we collapse lanes horizontally, as we can accomodate more lanes in the viewport

**Comment by SaiPavanManda** [comment]
@harshaktg you can achieve that using CSS and some conditional rendering using a custom lane header, according to my requirement.

**Comment by harshaktg** [comment]
Yes @SaiPavanManda , we can achieve them. But it will be much easier if we had an additional prop in that component to achieve the same.
